.github/workflows/pypi-publish.yml

```diff
@@ -14,29 +15,8 @@ on:
       - v*
 
 jobs:
-  Publish:
-    runs-on: ubuntu-20.04
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Set up Python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install setuptools wheel babel
-
-      - name: Build package
-        run: python setup.py sdist bdist_wheel
-
-      - name: Publish on PyPI
-        uses: pypa/gh-action-pypi-publish@v1.3.1
-        with:
-          user: __token__
-          # The token is provided by the inveniosoftware organization
-          password: ${{ secrets.pypi_token }}
+  build-n-publish:
+    uses: inveniosoftware/workflows/.github/workflows/pypi-publish.yml@master
+    secrets: inherit
+    with:
+      babel-compile-catalog: true
```

Comment:
`babel-compile-catalog` is only necessary if a `translation` directory exists, and the whole action can fail if the directory doesn't exists